src/components/CellViews/StringCell.js

```diff
@@ -4,7 +4,8 @@ const valueToString = (value) => {
   try {
     return JSON.stringify(value);
   } catch {
-    return value.toString();
+    /* if undefined or null we'll treat it as empty string, otherwise toString(), this will allow 0 and false to render properly */
+    return (value ?? "").toString();
```

Comment:
the same comment as in https://github.com/HumanSignal/dm2/pull/295:
> what kind of value could be here? it won't fail on null or undefined and that's the only values triggering nullish coalescing.

the issue with `false` and `0` is fully fixed by next change. but most probably it won't work as expected if you want to return empty string for `null` and `undefined`, then you have to check for them specifically.

Reply:
I applied a change based on our discussion. i also added a unit test in LS PR